integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -391,6 +391,24 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
         tableProperties.get(CarbonCommonConstants.CACHE_LEVEL).get,
         tableProperties)
     }
+    // validate long_string_columns from tale properties
+    var longStringColumns = varcharColumns.map(column => column.toUpperCase)
```

Comment:
change to `varcharColumns.map(toUpperCase)`, same for next line

Reply:
ok